src/backend/electron/main.ts

```diff
@@ -754,7 +754,13 @@ registerIpcMainHandle<IpcMainHandle>({
       win.maximize();
     }
   },
-
+  TOGGLE_FULLSCREENMODE: () => {
```

Comment:
```suggestion
  TOGGLE_FULLSCREEN: () => {
```
の方がいいかも？（これを変えるなら他のところも変える必要があるはず）

Reply:
たしかに`FULLSCREENMODE`一単語はほんのちょっぴり違和感あるかもですね！
`TOGGLE_FULLSCREEN_MODE`という手もありそう。